src/Writer/AbstractWriterMultiSheets.php

```diff
@@ -81,6 +82,37 @@ final public function setCurrentSheet(Sheet $sheet): void
         $this->workbookManager->setCurrentSheet($sheet);
     }
 
+    /**
+     * Row coordinates are indexed from 1, columns from 0 (A = 0),
+     * so a filter B2:G2 looks like $writer->setAutoFilterForCurrentSheet(1, 2, 6, 2);.
+     *
+     * @param 0|positive-int $fromColumnIndex
+     * @param positive-int   $fromRow
+     * @param 0|positive-int $toColumnIndex
+     * @param positive-int   $toRow
+     */
+    public function setAutoFilterForCurrentSheet(
```

Comment:
No need to add a new API to `AbstractWriterMultiSheets`, adding it to `Sheet` should suffice